### PR TITLE
first-branch

### PR DESCRIPTION
**Pull request by alexander-morrison**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -33,7 +33,49 @@ const App = () => {
...
+          return eachWord + "way"
+        } else if (eachWord[0] === 'i') {
+          return eachWord + "way"
+        } else if (eachWord[0] === 'o') {
+          return eachWord + "way"
+        } else if (eachWord[0] === 'u') {
+          return eachWord + "way"
+        }
+      } 
+      return firstVowel()
```
Comment (WalkerTexas-coder) [inline src/App.js]: This return ends the function. First vowel doesn't need to be found using a function if it's already available via vowelsArray[0]

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
This code should cause an issue

File: src/App.js (hunk 1 above)
Comment (kjv-engineer) [reply src/App.js]: We haven't figured out how to access vowels array and make it work. We hard coded it and it worked. We want to make it work with vowels array but still trying.

Comment (kjv-engineer) [reply src/App.js]: nevermind

**Review by WalkerTexas-coder — APPROVED** [review]

**Pull request merged**